nodeJS/express_and_mongoose/express_102_crud_and_mvc.md

```diff
@@ -30,7 +30,7 @@ The CRUD operations roughly correlate to the HTTP methods that you can employ in
 
 MVC is another common concept in web development and also something that is likely to come up in an interview question.  MVC stands for _Model, View, Controller_ and refers to the architecture of your code. Basically, it is a way to organize your application by separating all of the actions into 3 main components: Models, Views and Controllers.
 
-<span id="model">**Models**</span> are the basic building blocks of your database. So for every entry in your DB (books, authors, etc. in our Library Project), you'll create a model that holds the details of that entry. Models define the types of information that get used by your views and controllers.
+<span id="model">**Models**</span> are the basic building blocks of your database. So for every type of entries in your DB (books, authors, etc. in our Library Project), you'll create a model that holds the details of that entry. Models define the types of information that get used by your views and controllers.
```

Comment:
I am stuck on "entries" being plural and it doesn't quite sound right to me.  What do you think of using "type of entry"?

Reply:
I thought about this too 😅 
I appears both are correct
![image](https://github.com/TheOdinProject/curriculum/assets/55961349/470d4a46-08d6-4d02-87dc-44308b73785f)
whatever sounds better for you,
 if we change it to be singular might we also change `(books, authors, etc. in our Library Project)` to (book, author, etc. in our Library Project)? (it's why I ended up choosing plural)

Reply:
I noticed this sentence `you'll create a model that holds the details of that entry` should "entry" here be changed to "type" or "type of entry?

Reply:
Good catch. That sentence does need some updating. I am open to suggestions, but in addition to adding "type of entry" to it, I believe "will hold" reads better then just "holds"

> Models are the basic building blocks of your database. So for every type of entry in your DB (book, author, etc. in our Library Project), you'll create a model that will hold the details of that type of entry. Models define the types of information that get used by your views and controllers.

Reply:
agreed!